src/main/java/stsjorbsmod/cards/cull/deckoftrials/RepressedMemory.java

```diff
@@ -0,0 +1,74 @@
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.GainEnergyAction;
+import com.megacrit.cardcrawl.cards.CardQueueItem;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.relics.RunicPyramid;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.actions.DecreaseMaxHpAction;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+import stsjorbsmod.patches.ExertedField;
+import stsjorbsmod.patches.SelfExertField;
+import stsjorbsmod.powers.RepressedMemoryPower;
+
+import java.util.Arrays;
+
+public class RepressedMemory extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(RepressedMemory.class);
+
+    private static final CardRarity RARITY = CardRarity.SPECIAL;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 0;
+    private static final int MAX_HP_LOSS = 1;
+    private static final int ENERGY_GAIN = 2;
+    private static final int CARD_DRAW = 3;
+    private static final int UPGRADE_MAX_HP_LOSS_AMOUNT = -1;
+
+    public RepressedMemory() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        magicNumber = baseMagicNumber = CARD_DRAW;
+        urMagicNumber = baseUrMagicNumber = ENERGY_GAIN;
+        metaMagicNumber = baseMetaMagicNumber = MAX_HP_LOSS;
+    }
+
+    @Override
+    public void upgrade() {
+        if (!upgraded) {
+            upgradeName();
+            upgradeMetaMagicNumber(UPGRADE_MAX_HP_LOSS_AMOUNT);
+            upgradeDescription();
+        }
+    }
+
+    @Override
+    public void use(AbstractPlayer abstractPlayer, AbstractMonster abstractMonster) {
+        addToBot(new DrawCardAction(this.magicNumber));
+        addToBot(new GainEnergyAction(this.urMagicNumber));
+        if (!SelfExertField.selfExert.get(this) || !ExertedField.exerted.get(this)) {
+            addToBot(new ApplyPowerAction(abstractPlayer, abstractPlayer, new RepressedMemoryPower(abstractPlayer, this)));
+        }
+    }
+
+    @Override
+    public void triggerOnManualDiscard() {
+        this.addToBot(new DecreaseMaxHpAction(AbstractDungeon.player, AbstractDungeon.player, this.metaMagicNumber, AbstractGameAction.AttackEffect.POISON));
+    }
+
+    @Override
+    public void triggerOnEndOfPlayerTurn() {
+        if (AbstractDungeon.player.hand.group.contains(this)) {
+            if (!AbstractDungeon.player.hasRelic(RunicPyramid.ID)) {
+                addToBot(new DecreaseMaxHpAction(AbstractDungeon.player, AbstractDungeon.player, this.metaMagicNumber, AbstractGameAction.AttackEffect.POISON));
```

Comment:
I thought `whenever this card is discarded` was for manual discard and not end of turn discard. I'm thinking Reflex and Tactician, though the wording on there is `If this card is discarded from your hand,...`

Reply:
Checked with jorbs because I was unsure, I think this is intended behavior. 

See: https://discordapp.com/channels/329758756076519436/638465421598654496/771371781217976330

Reply:
Don't mind me, just including the message as quote
>LankYesterday at 9:56 AM
> `@jorbs` Repressed Memory: 0: Draw 3 cards, gain 2 energy. Exert at end of fight. Lose 1 max HP when discarded.  --- Do we lose Max HP if this is discarded at the end of your turn if it's not played?
> jorbsYesterday at 9:59 AM
> yes